blog/2024-04-18-opentofu-1-7-0-beta1.md

```diff
@@ -0,0 +1,225 @@
+
+:::warning
+
+Do not test this release on a production project! It is not a stable release!
+
+:::
+
+## Downloading the beta release
+
+The beta release is available exclusively from the [GitHub Releases page](https://github.com/opentofu/opentofu/releases/tag/v1.7.0-beta1). Please select the appropriate file for your platform. Here are some quick links:
+
+| Platform/Device                                                                 | Download link                                                                                                                                    |
+|---------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Desktop Windows computer**<br />(64-bit)                                      | [tofu_1.7.0-beta1_windows_amd64.zip](https://github.com/opentofu/opentofu/releases/download/v1.7.0-beta1/tofu_1.7.0-beta1_windows_amd64.zip)     |
+| **MacOS**<br />(Macbook M1 or higher; ARM64)                                    | [tofu_1.7.0-beta1_darwin_arm64.tar.gz](https://github.com/opentofu/opentofu/releases/download/v1.7.0-beta1/tofu_1.7.0-beta1_darwin_arm64.tar.gz) |
+| **MacOS**<br />(Macbook pre-M1 or lower; AMD64)                                 | [tofu_1.7.0-beta1_darwin_amd64.tar.gz](https://github.com/opentofu/opentofu/releases/download/v1.7.0-beta1/tofu_1.7.0-beta1_darwin_amd64.tar.gz) |
+| **Intel/AMD Linux computer or server**<br />(AMD64)                             | [tofu_1.7.0-beta1_linux_amd64.tar.gz](https://github.com/opentofu/opentofu/releases/download/v1.7.0-beta1/tofu_1.7.0-beta1_linux_amd64.tar.gz)   |
+| **ARM-based Linux computer<br />or<br />Raspberry Pi 3 or higher**<br />(ARM64) | [tofu_1.7.0-beta1_linux_arm64.tar.gz](https://github.com/opentofu/opentofu/releases/download/v1.7.0-beta1/tofu_1.7.0-beta1_linux_arm64.tar.gz)   |
+
+For the releases above, please unpack the archive and you should find the `tofu` binary inside. You can also use the [standalone installer](https://opentofu.org/docs/intro/install/standalone/) to download the release with signature verification.
+
+## Provider-defined functions
+
+The new Terraform provider SDK adds support for provider-defined functions you can use directly in OpenTofu. This is a significant improvement over using data sources as provider-defined functions don't increase the size of your state file and require less code to write.
+
+To test, you can use the [corefunc](https://github.com/northwood-labs/terraform-provider-corefunc) provider by Ryan Parman:
```

Comment:
I wonder if we link to the docs for this function too, like this page? https://library.tf/providers/northwood-labs/corefunc/latest